.github/workflows/ci.yml

```diff
@@ -202,3 +202,33 @@ jobs:
 
     - name: Self build
       run: npx hereby build-src --built
+
+  unused-baselines:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: "*"
+        check-latest: true
+    - run:  npm ci
+
+    - name: Remove all baselines
+      run: rm -rf tests/baselines/reference
+
+    - name: Run tests
+      run: npm test &> /dev/null || exit 0
+
+    - name: Accept baselines
+      run: npx hereby baseline-accept
+
+    - name: Run tests again and print unused baselines
+      run: |
+        if ! git diff --exit-code --quiet; then
+          echo "::group::npm test"
+          npm test
```

Comment:
This extra test ensures that the tests still pass after the baselines are removed. But, I think I'm going to drop this, because theoretically anyone who fails this pass will delete the unused baselines and then push their code up, having it rerun.

Reply:
(This was really only a concern when we still had baselines that _weren't_ produced by `npm test`, e.g. the docker and user baselines, which I have since removed.)